piksi_tools/console/console.py

```diff
@@ -190,6 +210,25 @@ class SwiftConsole(HasTraits):
   update_view = Instance(UpdateView)
   log_level_filter = Enum(list(SYSLOG_LEVELS.itervalues()))
 
+
+########## TRY TO IMPLEMENT FIX TYPE AND #SATS HERE #########
+  
+  mode = Str('') 
+  num_sats = Int(0)
+  port = Str('')
+
+  running = Bool(True)
+  logging_button=Button('Log')
+  #logging = Bool(False)
+
+  file_name = File
+
+  CSV_or_JSON = Bool
```

Comment:
change the name of this variable as it's not an "either or " propositoin to have json or csv logging